app/models/user.rb

```diff
@@ -20,6 +22,14 @@ def has_all?
     identities.pluck("provider").count == 2
   end
 
+  def photo
+    if (avatar.url == "original/default.png") && has_google? 
```

Comment:
I would check `paperclip` docs. I bet they have a method to check if `avatar` is missing.